Tutorial.md

```diff
@@ -11,477 +12,247 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
+```
 
-                Link: <http:/api.gistfox.com/>;rel="self",<http:/api.gistfox.com/gists>;rel="gists"
+## Metadata
 
-        + Body
-
-                {
-                    "_links": {
-                        "self": { "href": "/" },
-                        "gists": { "href": "/gists?{since}", "templated": true }
-                    }
-                }
+At the very top of this Blueprint, you will find a metadata section. In this case we have specified that `FORMAT` has the value of `1A`. The format keyword is used to specify the version of the API Blueprint. In this case 1A.
 
-What is going on here?
+## API Name & Description
 
-#### Resource
+After the metadata we have a heading, headings start with a hash (#) followed by a title. In this case `# Polls`. You can use up to 6 hashes which allows you to have sub-headings. The first heading in the Blueprint document serves as the API Name.
 
-    # Gist Fox API Root [/]
+Afterwards, there is a description of the Polls API, this mentions what the API is used for. You can use further headings to break up the description sections.
 
-In API Blueprint a resource definition starts with a Markdown header with the resource name followed by its URI enclosed in square brackets. The URI is relative to the API root. So in this case it is just `/`.
+## Resource Groups
 
-> **NOTE:** Resource name is an [API Blueprint Identifier][] and as such it can be composed only from a combination of alphanumerical characters, underscores, dashes and a spaces.
+Now it's time to start documenting the resources inside of the API. To start of with, the `Group` keyword is used inside a heading to specify a group of related resources.
 
-#### Resource Description
+```markdown
+# Group Questions
 
-    # Gist Fox API Root [/]
-    Gist Fox API entry point.
+Resource related to questions in the API.
+```
 
-    This resource does not have any attributes. Instead it offers the initial API affordances in the form of the HTTP Link header and haveL links.
+## Resource
 
-As with the API Name the resource name can be followed by an arbitrary Markdown formatted description.
+Inside of the questions resource group, we have a resource called "Questions collection". This resource allows you to view a list of questions. The heading specifies the URI used to access the resource inside of square brackets at the end of the heading.
 
-#### Action
+```markdown
+## Questions collection [/questions]
+```
 
-    ## Retrieve Entry Point [GET]
+### Actions
 
-Here we have just defined an action named "Retrieve Entry Point" utilizing the `GET` [HTTP Request Method][]. As with a resource and API name we can add any arbitrary description here but since the action name is pretty self explanatory we will skip it for now.
+API Blueprint allows you to specify the actions you can make on each resource. A sub-heading inside of the resource with a title, followed by the HTTP method to use inside the square brackets.
```

Comment:
The phrase quoted by @zdne is not a complete sentence. I also prefer "within" to inside as well, but that's not a hard rule. Additionally, I think it would be good to point out an example of what you are describing is below this paragraph.